pkg/raft/rafttest/interaction_env_handler.go

```diff
@@ -47,7 +47,8 @@ func (env *InteractionEnv) Handle(t *testing.T, d datadriven.TestData) string {
 	case "add-nodes":
 		// Example:
 		//
-		// add-nodes <number-of-nodes-to-add> voters=(1 2 3) learners=(4 5) index=2 content=foo async-storage-writes=true
+		// add-nodes <number-of-nodes-to-add> voters=(1 2 3) learners=(4 5) index=2
+		// content=foo async-storage-writes=true crdb-version=20
```

Comment:
Is 20 in the `crdb-version=20` a stable ID? Would it change when version gate consts are removed/replaced when we roll the code to the next CRDB version?